pkg/gofr/datasource/mongo/mongo.go

```diff
@@ -116,6 +181,12 @@ func (c *Client) Connect() {
 
 	mongoBuckets := []float64{.05, .075, .1, .125, .15, .2, .3, .5, .75, 1, 2, 3, 4, 5, 7.5, 10}
 	c.metrics.NewHistogram("app_mongo_stats", "Response time of MONGO queries in milliseconds.", mongoBuckets...)
```

Comment:
```suggestion
 	c.metrics.NewHistogram("app_mongo_stats", "Response time of MongoDB queries in milliseconds.", mongoBuckets...)
```

Minor typo that could be fixed even if out of the PR